addon/authenticators/osf-cookie.js

```diff
@@ -36,7 +36,7 @@ export default Base.extend({
             // Push the result into the store for later use by the current-user service
             // Note: we have to deepcopy res because pushPayload mutates our data
             // and causes an infinite loop because reasons
-            this.get('store').pushPayload(Ember.copy(res, true));
+            this.get('store').pushPayload(Ember.copy(res['meta']['current_user'], true));
```

Comment:
If they're not logged in, `res.meta.current_user` will be missing/null. Might be better not to `pushPayload` in that case